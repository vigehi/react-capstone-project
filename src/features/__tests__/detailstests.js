/* eslint-disable no-unused-vars */
import { Provider } from 'react-redux';
import { render, screen } from '@testing-library/react';
import Home from '../home';
import store from '../../app/store';

describe('Test specific post', () => {
  it('renders correctly', () => {
    const post = [
      {
        id: 2424923,
        info: 'Zombie startups',
      },
    ];

    const { container } = render(
      <Provider store={store}>
        <Home />
      </Provider>,
    );
    const message = screen.getByText('Enter a valid post!');
    expect(message).toMatchSnapshot();
  });
});
