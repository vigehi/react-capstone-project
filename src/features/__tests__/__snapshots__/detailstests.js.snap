// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test specific post 1`] = `
<div
  class="heading"
>
  Enter a post
</div>
`;
